server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+    // insertFStreetCommunityCenterData();
+    // insertSoutheastCommunityCollegeData();
+	die('SUCCESS');
+}
+
+// Need to create the new sites
+// Create Shifts for each site
+// Create appointment times within those shifts
+// Create role limits for that site/shift/role
+
+function insertAndersonLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The AL data has already been inserted');
+	}
+
+	$siteId = 2; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Wednesdays
+		$dates = getWeeklyDatesFromRange('2020-01-29', '2020-04-08');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 20:00:00", $siteId);
+
+            // Appointment numbers are from per Katie's parameter sheet
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 9, 9, 100, 60, $siteId);
+            $secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 9, 9, 100, 60, $siteId);
+            $thirdAppointmentTimeId = insertAppointmentTime("$date 19:00:00", 5, 5, 100, 60, $siteId);
+        }
+        
+        // Thursdays
+        $thursdayDates = getWeeklyDatesFromRange('2020-01-30', '2020-04-09');
+        foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 20:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 6, 6, 100, 60, $siteId);
+            $secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 6, 6, 100, 60, $siteId);
+            $thirdAppointmentTimeId = insertAppointmentTime("$date 19:00:00", 5, 5, 100, 60, $siteId);
+        }
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Anderson Library data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertCenterForPeopleInNeedData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Center for People in Need data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+        $siteId = 5; // Manually obtained from PROD DB
+
+		// Tuesdays
+		$dates = getWeeklyDatesFromRange('2020-02-04', '2020-03-24');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 11:00:00", "$date 14:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 4, 4, 100, 60, $siteId);

```

Comment:
We should verify with Katie that we want to absolutely cap the number of appointments at the number she's specified in the parameter document. There might be a chance that she actually doesn't want to.

Reply:
Did not change. "So even if 0 preparers signed up or 50 preparers signed up, we'd have 8 appointments during that time slot. Is this what you were expecting? YES"